vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
+```
+        - name: SPRING_PROFILES_ACTIVE
+          value: default,swagger
+```
 
-The Entando core exposes REST APIs for every action that can be taken in
-the App Builder environment. For example, you can use
-these APIs to create pages, create page templates or to add widgets to
-pages. The APIs can be used to support automation, testing, or
-integrations with external systems.
+5. (Optional) Reset the deployment `spec.replicas` back to 1 if it was scaled to "0" in the previous steps. Save the deployment to update.
 
-### API structure
+### Disable Swagger UI
 
-All of the APIs share a common top level structure. Each response will
-contain a top level entry for `errors`, `metadata`, and `payload`.
+Repeat the steps above, but in step 4, remove `swagger` from the value list.
 
-The `errors` will always contain code and a message string indicating an
-error condition in the request. The `metadata` section is used for
-paging, sorting, filtering and data that is distinct from the body. The
-body of each response is included in the `payload` section of the
-response and varies according to each API.
+## Find Your Client Secret
+You'll need your client credentials to execute the Entando APIs. 
 
-### Models
+1. Login into your Keycloak Administration Console. To find the Kubernetes credentials, see the [Entando Identity Management System](./identity-management.md) page.
 
-All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
+2. From the left nav, go to `Clients`
 
-## Tutorial
+3. Select the desired client (e.g. in a quickstart environment this is `quickstart`)
 
-1. Access your application Swagger UI as discussed above
+4. Click on the `Credentials` tab to retrieve the secret 
 
-2. Click on the `Authorize` button in the upper right corner
+## Access the APIs on Swagger
+1.  To see the APIs, go to: 
+``` sh
+http://[YOUR-HOST-NAME]/entando-de-app/api/swagger-ui.html
+```
 
-3. Enter the client id and client secret in the open window and click `Authorize`
+2. Click on the Authorize button in the upper right corner.
 
-4. If you are redirected to the Entando login page, log in with your credentials (default are `admin`/`adminadmin`)
+3. Enter the client id and secret and click Authorize.
 
-5. You will be redirected to the Swagger UI page, now authenticated
+4. You will be prompted to log in to your Keycloak instance as an admin user if you have not already done so. The defaults are admin/adminadmin. 
 
-6. Use the **Try it out** button on the APIs
+5. You will be redirected to the authenticated Swagger UI page. Select an API to see the methods in the drop-down list and click the `Try it out` button.  
```

Comment:
**its** methods (not all methods are the same??)

Reply:
no no no, sorry - i mean do the same methods apply to all the APIs or do different APIs support different methods? if the same always apply then "the methods" is the way to go because there is only one set and it's used for every API - the specificity works. but if different APIs have different methods then "the" APIs is imprecise, but "its APIs" is specific

Reply:
not sure what the GET method has to do with this? pretty sure most everyone would use "its"
<img width="719" alt="Screen Shot 2023-02-17 at 8 50 08 AM" src="https://user-images.githubusercontent.com/93157274/219714476-28ebf00d-1e52-439b-8c71-09b11068342e.png">

Reply:
great. that clears it up